tests/test_network_utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import pytest
+import requests_mock
+
+from kf_lib_data_ingest.config import VERSION, PROJECT_GITHUB_URL
+from kf_lib_data_ingest.network.utils import get_tag_commit_sha
+
+
+def test_get_tag_commit_sha(caplog):
+    """
+    Test get_tag_commit_sha
+    """
+    caplog.set_level(logging.DEBUG)
+
+    # Test successful get
+    assert get_tag_commit_sha(VERSION)
+    assert 'Got tags' in caplog.text
+
+    # Test get tag that doesn't exist
+    with pytest.raises(Exception) as e:
+        get_tag_commit_sha('foo')
+        assert 'Could not find tag' in str(e)
```

Comment:
unindent the assert

Reply:
But its checking for the message in the exception msg

Reply:
Doesn't matter. If the assert fails it will get swallowed.

Reply:
I guess you want something like:
```
with pytest.raises(Exception):
      try:
          get_tag_commit_sha('foo')
      except Exception as e:
          emsg = str(e)
          raise
assert 'Could not find tag' in emsg
```